flexeval/core/evaluate_chat_response.py

```diff
@@ -225,6 +225,8 @@ def evaluate_chat_response(  # noqa: C901
             "references": output["chat_instance"].references,
             **output["metrics"],
         }
+        if output["lm_output"].raw_text:
+            restructured_output["raw_lm_output"] = output["lm_output"].raw_text
```

Comment:
Note: Similar logic was available sometime in the past, but it's missing.
This PR restores it.

https://github.com/sbintuitions/flexeval/blob/c2c3eb20858906cef5e144ac740543a47f0fa81f/flexeval/core/evaluate_generation.py#L106